src/components/common/Select/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+import "./Select.scss";
+import { UseFormRegister, FieldValues } from "react-hook-form";
```

Comment:
In general the convention is to import the third party libraries before the styles